src/main/java/com/uid2/shared/model/Site.java

```diff
@@ -82,11 +134,11 @@ public boolean equals(Object o) {
         if (this == o) return true;
         if (o == null || getClass() != o.getClass()) return false;
         Site site = (Site) o;
-        return id == site.id && name.equals(site.name) && enabled.equals(site.enabled) && domainNames.equals(site.domainNames) && clientTypes.equals(site.clientTypes) && created == site.created;
+        return id == site.id && name.equals(site.name) && Objects.equals(description, site.description) && enabled.equals(site.enabled) && domainNames.equals(site.domainNames) && clientTypes.equals(site.clientTypes) && visible.equals(site.visible) && created == site.created;
```

Comment:
I guess we don't need the Objects.equals here anymore then since we're defaulting to ""?

Reply:
yep good point - done